pkg/controller/operators/olm/operator_test.go

```diff
@@ -3576,6 +3576,10 @@ func TestSyncOperatorGroups(t *testing.T) {
 				}
 			}
 
+			// Sync again to catch provided API changes
+			err = op.syncOperatorGroups(tt.initial.operatorGroup)
```

Comment:
Needed because OperatorGroup sync returns after each meaningful update and relies on the update event to trigger another sync.